src/compiler/postprocessor.rs

```diff
@@ -71,12 +71,12 @@ impl PostProcessor {
     pub fn get_default() -> Vec<Self> {
         let mut postprocessors = Vec::new();
 
-        let shfmt = PostProcessor::new("shfmt", "/usr/bin/shfmt");
+        let shfmt = PostProcessor::new("shfmt", "shfmt");
```

Comment:
Could even remove the duplicate argument to `Postprocessor::new()`, given you're now passing the same thing twice. But check with @b1ek as it's her code.

Reply:
the first is the human readable name, second is the executable name